web/src/views/SubmissionPortal/SubmissionView.vue

```diff
@@ -1,5 +1,6 @@
 <script lang="ts">
 import {
+  computed,
   defineComponent, PropType, toRef, watch,
```

Comment:
```suggestion
  computed,
  defineComponent, 
  PropType, 
  toRef, 
  watch,
```

If you could just make the import statement's style more consistent I'd appreciate that.